server/src/fights/fights.service.spec.ts

```diff
@@ -23,186 +26,246 @@ describe('FightsService', () => {
 
     redEventsHistory: [],
     blueEventsHistory: [],
+
+    timer: new Timer(1),
   };
 
   beforeAll(async () => {
     app = await Test.createTestingModule({
       providers: [FightsService],
     }).compile();
+    fightService = app.get(FightsService);
   });
 
   describe('newFight', () => {
     it('should create new fight and find it', () => {
-      const fightService = app.get(FightsService);
       fightService.newFight(fight);
```

Comment:
Wywołanie newFight powinno znaleźć się w metodzie beforeAll. Nie ma gwarancji, że ten test zostanie wykonany jako pierwszy (bo np. w przyszłości zmieni się kolejność testów) i w takm wypadku pozostałe testy, które zakładają już dodaną przynajmniej jedną walkę, nie przejdą.

Reply:
Z drugiej strony wtedy wywołanie newFight nie będzie środku testu... Nie wiem, może na razie zostawmy tak, a jak się pojawią problemy to zmodyfikujemy ten fragment